api/src/modules/auth/controllers/auth.ctrl.ts

```diff
@@ -14,6 +14,8 @@ interface AuthReq {
   };
 }
 
+export const TOKEN_TYPE = 'Bearer';
```

Comment:
I use this standard for top level constants: `const TokenType`. let's stay with one, 
whichever it be. if you really need this style, please update constant in entire project.